mappings/net/minecraft/world/biome/source/util/OverworldBiomeParameters.mapping

```diff
@@ -0,0 +1,122 @@
+		ARG 1 temperature
+		ARG 2 humidity
+		ARG 3 weirdness
+	METHOD m_fgargpsb getPeaksAndValleysDescription (D)Ljava/lang/String;
+		ARG 0 erosion
+	METHOD m_gaoknfao addUndergroundBiomeTo (Ljava/util/function/Consumer;Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;FLnet/minecraft/unmapped/C_hcaelhhx;)V
+		ARG 1 parameters
+		ARG 2 temperature
+		ARG 3 humidity
+		ARG 4 continentalness
+		ARG 5 erosion
+		ARG 6 weirdness
+		ARG 7 offset
+		ARG 8 biome
+	METHOD m_gwliomzb getContinentalnessDescription (D)Ljava/lang/String;
+		ARG 1 continentalness
+	METHOD m_iyrqfmug pickSlopeBiome (IILnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;)Lnet/minecraft/unmapped/C_hcaelhhx;
+		ARG 1 temperature
+		ARG 2 humidity
+		ARG 3 weirdness
+	METHOD m_lbpflehx pickShatteredCoastBiome (IILnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;)Lnet/minecraft/unmapped/C_hcaelhhx;
+		ARG 1 temperature
+		ARG 2 humidity
+		ARG 3 weirdness
+	METHOD m_ligkrksk getHumidityDescription (D)Ljava/lang/String;
+		ARG 1 humidity
+	METHOD m_lombgnay pickBiomeOrShatteredSavanna (IILnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;Lnet/minecraft/unmapped/C_hcaelhhx;)Lnet/minecraft/unmapped/C_hcaelhhx;
+		ARG 1 temperature
+		ARG 2 humidity
+		ARG 3 weirdness
+		ARG 4 biome
+	METHOD m_ntbginte addInlandBiomesTo (Ljava/util/function/Consumer;)V
+		ARG 1 parameters
+	METHOD m_pzplxltp addLowSliceTo (Ljava/util/function/Consumer;Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;)V
+		ARG 1 parameters
+		ARG 2 weirdness
+	METHOD m_riujaelq getErosionDescription (D)Ljava/lang/String;
+		ARG 1 erosion
+	METHOD m_sltsvbux addPeaksTo (Ljava/util/function/Consumer;Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;)V
+		ARG 1 parameters
+		ARG 2 weirdness
+	METHOD m_tgqopzcf pickBadlandsBiome (ILnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;)Lnet/minecraft/unmapped/C_hcaelhhx;
+		ARG 1 humidity
+		ARG 2 weirdness
+	METHOD m_tgwqmgjx pickRegularBiome (IILnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;)Lnet/minecraft/unmapped/C_hcaelhhx;
+		ARG 1 temperature
+		ARG 2 humidity
+		ARG 3 weirdness
+	METHOD m_unymhsao getNoiseValueDescription (D[Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;)Ljava/lang/String;
+		ARG 0 value
+		ARG 2 parameters
+	METHOD m_vqgtadov pickPlateauBiome (IILnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;)Lnet/minecraft/unmapped/C_hcaelhhx;
+		ARG 1 temperature
+		ARG 2 humidity
+		ARG 3 weirdness
+	METHOD m_xikbagsm addSurfaceBiomeTo (Ljava/util/function/Consumer;Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;FLnet/minecraft/unmapped/C_hcaelhhx;)V
```

Comment:
Just `addBiomeTo` here? As it's not just for surface biomes, but actually in almost the entire depth range.

Reply:
Not entirely sure this adds 2 times the biome with depth 0 and depth 1 separately.
Why the underground one is added one time with depth 0.1 -> 0.9

Reply:
I assume that's because the cave biome one would otherwise be completely inside the surface biome, which would mean either the surface one or the cave one would never generate at that point.

Reply:
This also follows the mojmap name